code/modules/credits/credits.dm

```diff
@@ -126,6 +127,8 @@ var/global/datum/credits/end_credits = new
  */
 /datum/credits/proc/on_round_end()
 	draft()
+	if(!do_not_change_credits_song)
```

Comment:
Double negative in this context is confusing, make it a `change_credits_song` and invert the boolean checks